src/aics-image-viewer/assets/styles/variables.css

```diff
@@ -1,14 +1,16 @@
-$app-height: 100%;
-$dark-gray: #313131;
-$purple: #8048f3;
-$highlight-purple: #8950d9;
-$light-gray: #6e6e6e;
-$bright-green: #b2d030;
-$background-gray: #fafafa;
-$header-gray: #4b4b4b;
-$background-dark-gray: #131313;
+/* TODO: Replace all of these with CSS variables (e.g. var(--color-text-body) and remove PostCSS from the project.*/
+
+/* $app-height: 100%; */
+/* $dark-gray: #313131; */
+/* $purple: #8048f3; */
+$highlight-purple: #9e6aff;
+/* $light-gray: #6e6e6e; (DUPLICATE) */
+/* $bright-green: #b2d030; */
+/* $background-gray: #fafafa; */
+/* $header-gray: #4b4b4b; (DUPLICATE) */
+/* $background-dark-gray: #131313; */
```

Comment:
ok so I really don't understand .css files, but why do we have all of these commented out colors here? Is this just defining our color scheme and you'd uncomment these lines if you needed the relevant color?

Reply:
These are commented out because I wanted to mark which ones were unused; I want to go through in the future and delete all of these variables. I expect this whole file to be deleted in the near future, but I can also delete these now.